### DISCUSSION TITLE
Test timeout has no effect

### DISCUSSION THREAD

**Discussion by flachica** [OCA/odoo-community.org#82] [Q&A] [answered]
I have seen the guide to run front-end tests and according to it, when a step is defined, [you can indicate](https://www.odoo.com/documentation/16.0/es/developer/reference/backend/testing.html#javascript) in the "timeout" property a time that has to pass to continue with the execution of the next step.

Well, in my case this does not happen. All the steps are executed immediately. 

Is there any condition for this timeout to take effect?

By the way, I am testing a module made by me for the POS.

Thanks

PD: This is a copy of a [thread](https://www.odoo.com/es_ES/forum/ayuda-1/test-timeout-has-no-effect-221072) added in Odoo community help forum

**Comment by chienandalu** [comment, marked as answer]
I'd say the effect for that parameter is not a delay but a time to let you're step run until is considered failed (thus the term timeout). I think there's no specific step delay option but you can set it globally with `step_delay` when you invoke the tour from `start_tour` in your test.
